### PR TITLE
Various grammar and spelling fixes.

### PR DESCRIPTION
**Pull request by Coconutwarrior97**
## About The Pull Request

Fixes grammar/spelling related to the Space Dragon as outlined by #50602.

## Why It's Good For The Game

Grammar fixes are good.

## Changelog
:cl:

fix: Fixes grammar/spelling related to Space Dragon.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/space_dragon.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	ranged = TRUE
 	mouse_opacity = MOUSE_OPACITY_ICON
 	butcher_results = list(/obj/item/stack/ore/diamond = 5, /obj/item/stack/sheet/sinew = 5, /obj/item/stack/sheet/bone = 30)
-	deathmessage = "screeches as its wings turn to dust and it collapses on the floor, life estinguished."
+	deathmessage = "screeches as its wings turn to dust and it collapses on the floor, its life exstinguished."
```
Comment (spessman-007) [inline code/modules/mob/living/simple_animal/hostile/space_dragon.dm:47]: Should be extinguished

**Review by spessman-007 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/space_dragon.dm (hunk 1 above)
Comment (Coconutwarrior97) [reply code/modules/mob/living/simple_animal/hostile/space_dragon.dm]: Thanks for the catch!

**Comment by IndieanaJones** [comment]
I was going to get to around to this, but I certainly don't mind someone else doing it, either.  👍

**Comment by kriskog** [comment]
merge conflict

**Comment by AnturK** [comment]
Still conflicts @Coconutwarrior97

**Comment by nemvar** [comment]
Doesn't compile

File: code/modules/mob/living/simple_animal/hostile/space_dragon.dm [hunk 2]
```diff
@@ -350,7 +350,11 @@ mob/living/simple_animal/hostile/space_dragon/proc/dragon_fire_line(turf/T)
 			main_objective.completed = TRUE
 	sound_to_playing_players('sound/machines/alarm.ogg')
 	sleep(100)
+<<<<<<< HEAD
+	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds.  Evacuation of the remaining crew will begin immediately.", "Central Command Spacial Corps")
+=======
 	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds.  Evacuation of the remamining crew will begin immediately.", "Central Command Spatial Corps")
+>>>>>>> remotes/origin/master
```
Comment (kriskog) [inline code/modules/mob/living/simple_animal/hostile/space_dragon.dm:357]: You need to resolve this conflict. Remove everything but the correct line.

Comment (kriskog) [inline code/modules/mob/living/simple_animal/hostile/space_dragon.dm:357]:
```suggestion
	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds.  
```

File: code/modules/mob/living/simple_animal/hostile/space_dragon.dm [hunk 3]
```diff
@@ -350,7 +350,7 @@ mob/living/simple_animal/hostile/space_dragon/proc/dragon_fire_line(turf/T)
 			main_objective.completed = TRUE
 	sound_to_playing_players('sound/machines/alarm.ogg')
 	sleep(100)
-	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds.  Evacuation of the remamining crew will begin immediately.", "Central Command Spatial Corps")
+	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds.
```
Comment (kriskog) [inline code/modules/mob/living/simple_animal/hostile/space_dragon.dm:353]:
```suggestion
	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds. Evacuation of the remamining crew will begin immediately.", "Central Command Spatial Corps")
```

File: code/modules/mob/living/simple_animal/hostile/space_dragon.dm [hunk 4]
```diff
@@ -350,7 +350,7 @@ mob/living/simple_animal/hostile/space_dragon/proc/dragon_fire_line(turf/T)
 			main_objective.completed = TRUE
 	sound_to_playing_players('sound/machines/alarm.ogg')
 	sleep(100)
-	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds.  Evacuation of the remamining crew will begin immediately.", "Central Command Spatial Corps")
+	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds. Evacuation of the remamining crew will begin immediately.", "Central Command Spatial Corps")
```
Comment (kriskog) [inline code/modules/mob/living/simple_animal/hostile/space_dragon.dm:353]:
```suggestion
	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds. Evacuation of the remaining crew will begin immediately.", "Central Command Spatial Corps")
```

**Pull request merged**